shared/css/header.scss

```diff
@@ -137,17 +137,21 @@
     }
 
     .user_menu_arrow_down,
-    .user_menu_arrow_up {
+    .user_menu_arrow_up,
+    .create_menu_arrow_up,
+    .create_menu_arrow_down
```

Comment:
With so many classes at some point it's probably worth factoring to one common class for this attribute, but your call on whether appropriate...